test/framework/ressources/basic-auth-app-deployment.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: mei
+  labels:
+    group: test
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      group: test
+  template:
+    metadata:
+      labels:
+        group: test
+    spec:
+      containers:
+      - name: example-app
+        image: lp150189/mei:0.1.0
```

Comment:
Where does the code of this application live?

Reply:
https://github.com/huydinhle/mei. I wrote this app to test the basic auth stuff quickly. Let me know if you know any better candidate though.